### PR TITLE
Health check updates

### PR DESCRIPTION
**Pull request by leejacobson-moj**
## Overview

Updates health check endpoint to include all dependant services.

### CODE REVIEW

File: feature.env [hunk 1]
```diff
@@ -5,8 +5,8 @@ TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
 TOKEN_VERIFICATION_ENABLED=true
 
 EDUCATION_AND_WORK_PLAN_API_URL=http://localhost:9091
-PRISONER_SEARCH_API_URL=http://localhost:9091
-PRISON_REGISTER_API_URL=http://localhost:9091
+PRISONER_SEARCH_API_URL=http://localhost:9091/prisoner-search-api
+PRISON_REGISTER_API_URL=http://localhost:9091/prison-register-api
```
Comment (leejacobson-moj) [inline feature.env:9]: Need to namespace these APIs so wiremock doesn't confuse common mocked endpoints.

File: integration_tests/mockApis/common.ts
Comment (leejacobson-moj) [inline integration_tests/mockApis/common.ts:1]: Added this to a common file. It felt wrong adding effectively the same stub to every API.

File: server/config.ts [hunk 2]
```diff
@@ -84,6 +84,7 @@ export default {
       apiClientSecret: get('API_CLIENT_SECRET', 'clientsecret', requiredInProduction),
       systemClientId: get('SYSTEM_CLIENT_ID', 'clientid', requiredInProduction),
       systemClientSecret: get('SYSTEM_CLIENT_SECRET', 'clientsecret', requiredInProduction),
+      includeInHealthCheck: true,
```
[the remark this answers is not in the document]
Comment (nathanrussell-moj-digital) [reply server/config.ts:87]: Thats a great idea 👍

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**